objects/screen.c

```diff
@@ -241,6 +243,143 @@ screens_exist(void)
     return globalconf.screens.len > 0;
 }
 
+#ifdef XCB_RANDR_GET_MONITORS
```

Comment:
I'm not sure why would we want to do version-based check, if we can check capability directly.